src/components/common/button.tsx

```diff
@@ -23,6 +23,10 @@ const StyledButton = styled.button<{ variant?: string }>`
         props.variant && props.variant === 'tertiary'
             ? `background-color: ${props.theme.componentsTheme.buttonTertiaryBackgroundColor};`
             : ''}
```

Comment:
PP: Could you use default props here?, you can see an example [here](https://www.styled-components.com/docs/advanced). We are using the same in another part of the code (I did not found it sorry)